### ISSUE TITLE
Poor contrast for active tab with hidden file names on macOS .env

### ISSUE THREAD

**Issue by adampatterson** [arcticicestudio/nord-jetbrains#106]
This might be more me than the theme, but I can not see the `.env` file at all in the tab.  If there is a setting or an option to enable/disable that would be great. I was not able to find anything obvious.

I have File Colors disabled, and the related issues showed a fix when updated to 2019.1.3

![image](https://user-images.githubusercontent.com/31537/68088469-d7870400-fe1c-11e9-85dc-ca251b7b106b.png)

The contrast in the sidebar is fine.

![image](https://user-images.githubusercontent.com/31537/68088550-8deae900-fe1d-11e9-917f-a58d891446af.png)

![image](https://user-images.githubusercontent.com/31537/68088552-93483380-fe1d-11e9-9eaf-8dab583eac66.png)

PHP Storm Build 2019.2.4

**Comment by arcticicestudio** [comment]
Related to #99

---

Hi @adampatterson :wave:, thanks for your contribution :+1:

Looks like this is related (or almost a duplicate) of #99. The problem with VCS ignored files are unfortunately a well-known problem or better named a missing feature from the theme API to better handle situations where multiple color are combined.
I'm still trying to figure out how to solve this tricky design without breaking the theme's appearance and feeling while also keep track of reported problems to the Jetbrains bug tracker in order to fix this within the theme API itself.

**Comment by adampatterson** [comment]
Thanks for the quick reply,

Would it help to open an issue with JetBrains?

**Comment by DuboisS** [comment]
Any news ?

**Comment by arcticicestudio** [comment]
This is still an ongoing topic and there might be a solution by revisiting the current syntax theme file that might include definitions from the “old“ theme API that are now in conflict with the new theme API, but it requires more reproduction and testing.

**Comment by DroneZzZko** [comment]
Is there any open issue in JetBrains' issue tracker?

**Comment by DroneZzZko** [comment]
Found a workaround:

1. File | Settings | Version Control | **File Status Colors**
2. Select **Ignored** → Unselect **File Status Color** 

![image](https://user-images.githubusercontent.com/426416/97562690-b7b3fd00-19ea-11eb-9982-c61ffa84330b.png)
![image](https://user-images.githubusercontent.com/426416/97562927-0eb9d200-19eb-11eb-807a-a4b45e83f9ac.png)

**Comment by DuboisS** [comment]
Yes, already explain here: https://github.com/arcticicestudio/nord-jetbrains/issues/57 :)
